.appveyor.yml

```diff
@@ -21,7 +21,7 @@ before_build:
 
 build_script:
   - cmd: CompileSharpmake.bat Sharpmake.sln "%CONFIGURATION%" "%PLATFORM%"
-  - sh: msbuild -t:build -restore Sharpmake.sln /p:Configuration=\"$CONFIGURATION\" /p:Platform=\"$PLATFORM\" /nologo /v:m
+  - sh: sh CompileSharpmake.sh Sharpmake.sln "${CONFIGURATION}" "${PLATFORM}"
```

Comment:
Should invoke it with ./CompileSharpmake.sh, this will ensure that we have the +x flag

Reply:
On my Mac ./CompileSharpmake.sh is blocked ("permission denied"), to make it work I need to do a "chmod +x" before. I did it in 03df181a5f66d24acd99fb0165df758cf8c4a9f5